hibernate-core/src/main/java/org/hibernate/engine/jdbc/spi/SqlStatementLogger.java

```diff
@@ -20,9 +20,15 @@
  */
 public class SqlStatementLogger {
 	private static final Logger LOG = CoreLogging.logger( "org.hibernate.SQL" );
+	private static final Logger LOG_SLOW = CoreLogging.logger( "org.hibernate.SQL_SLOW" );
 
 	private boolean logToStdout;
 	private boolean format;
+	
+	/**
+	 * Configuration value that indicates slow query. (In milliseconds) 0 - disabled.
+	 */
+	private long logSlowQuery;
```

Comment:
Could you change this to a final field?

See also https://github.com/hibernate/hibernate-orm/commit/3483d7c02276b8d2d57929323b6715831235ce15

Reply:
Fixed in 35a2d98abd80e3ed64734e22f69ec6277b08726a